docs/using/administering/debugging.rst

```diff
@@ -23,6 +52,10 @@ They have unit names starting constructed with a ``flocker-`` prefix, e.g. ``flo
 
 It is possible to see the available unit names, and then view the logs with ``journalctl``:
 
+# FLOC-2647 Not sure if this example is helpful anymore
```

Comment:
Leave it here as a first step for monitoring Flocker, before getting to the stage of needing to report a bug. Although there is some repetition, a user might not realise that a section entitled Bug Reporting contains information that is useful when not intending to report a bug.

You can however change the first command to gather the unit names.